scripts/fabricstat

```diff
@@ -166,6 +222,24 @@ class FabricQueueStat(FabricStat):
             cnstat_dict[port_queue_name] = QueueStat._make(cntr)
         return cnstat_dict
 
+    def save_fresh_stats(self):
+        # Get stat for each port and save
+        counter_port_name_map = self.db.get_all(self.db.COUNTERS_DB, COUNTERS_FABRIC_PORT_NAME_MAP)
+        if counter_port_name_map is None:
```

Comment:
>counter_port_name_map [](http://example.com/codeflow?start=11&length=21)

not used. is it needed? if yes, do you want to print some info before return?

Reply:
printed out a message before the return